stingray/lightcurve.py

```diff
@@ -177,13 +186,14 @@ def __init__(self, time, counts, err=None, input_counts=True,
 
         self.err_dist = err_dist
 
-        self.tstart = self.time[0] - 0.5*self.dt
-        self.tseg = self.time[-1] - self.time[0] + self.dt
+        self.tstart = np.min(self.time) - 0.5*self.dt
```

Comment:
well, it's not if the user uses `sort_counts()`, because that will re-sort the time array by the number of counts per bin.